Makefile

```diff
@@ -23,43 +23,72 @@ MANDIR        ?= $(PREFIX)/share/man
 DOCDIR        ?= $(PREFIX)/share/doc
 SYSCONFDIR    ?= /etc
 CRONDIR       ?= $(SYSCONFDIR)/cron.d
+SYSTEMDDIR    ?= /lib/systemd/system
```

Comment:
The official path is _/usr/lib_, not _/lib_ - [from the docs](https://www.freedesktop.org/software/systemd/man/systemd.unit.html):
```
/etc/systemd/system/*
/run/systemd/system/*
/usr/lib/systemd/system/*
```
On many modern Linux, /lib is a symlink to /usr/lib now (depending on distro).

Reply:
My bad, I've fixed that in the latest commit.